supported_versions.md

```diff
@@ -0,0 +1,85 @@
+| Dependency | Minimum Version | Maximum Version | Auto-Instrumented |
+|------------|-----------------|-----------------|-----------------|
+| cloud.google.com/go/pubsub | v1.33.0 | v1.45.3 | true
+| github.com/99designs/gqlgen | v0.17.36 | v0.17.61 | true
+| github.com/IBM/sarama | v1.40.0 | v1.43.3 | true
+| github.com/Shopify/sarama | v1.38.1 | v1.43.3 | true
+| github.com/aws/aws-sdk-go | v1.44.327 | v1.55.5 | false
```

Comment:
```suggestion
| github.com/aws/aws-sdk-go | v1.44.327 | v1.55.5 | true
```

this one is auto-instrumented: https://github.com/DataDog/orchestrion/blob/main/internal/injector/builtin/yaml/cloud/aws-sdk.yml